src/StatusQ/Components/StatusChatList.qml

```diff
@@ -29,199 +31,262 @@ Column {
 
     signal chatItemSelected(string id)
     signal chatItemUnmuted(string id)
-    signal chatItemReordered(string id, int from, int to)
+    signal chatItemReordered(string categoryId, string id, int from, int to)
 
     onPopupMenuChanged: {
         if (!!popupMenu) {
             popupMenuSlot.sourceComponent = popupMenu
         }
     }
 
-    DelegateModel {
-        id: delegateModel
+    Column {
+        id: statusChatListItemsWrapper
+        width: parent.width
+        spacing: 4
```

Comment:
This column is needed to get a sense of how many items live in this chatlist.
At the moment, we always pass every chat item to every chat list and hide them if they don't belong to the corresponding category. This means, the model has never a `count: 0` so we have to rely on the `implicitHeight` of the `Repeater` that renders them.

We can't use the root `Column` for that because we have other elements living inside of it.